src/coreclr/vm/arm64/asmhelpers.S

```diff
@@ -951,23 +951,20 @@ LEAF_END Load_Stack_Ref, _TEXT
 .macro Copy_Ref argReg
     cmp x11, #16
     blt LOCAL_LABEL(CopyBy8\argReg)
+    sub x11, x11, #16
```

Comment:
We can simplify this even more by
* replacing the cmp by the sub
* replacing the blt target to go to the add x11, x11, #16
* removing the sub below the blt

Thank you for finding this issue!

Reply:
Thanks for the idea, added a new commit implementing this.